### PR TITLE
Fix the picard metrics header for multiqc compatability.

### PR DESCRIPTION
**Pull request by jasonwalker80**
Two PRs already exist to fix MultiQC:
https://github.com/ewels/MultiQC/pull/1114
https://github.com/ewels/MultiQC/pull/1120/

### CODE REVIEW

File: tasks/Qc.wdl [hunk 1]
```diff
@@ -171,7 +171,7 @@ task CollectRawWgsMetrics {
 
   Int memory_size = ceil((if (disk_size < 110) then 5 else 7) * memory_multiplier)
   String java_memory_size = (memory_size - 1) * 1000
-
+
```
Comment (tmooney) [inline tasks/Qc.wdl]: Git would be sad about this change.

File: tasks/Qc.wdl [hunk 2]
```diff
@@ -185,6 +185,10 @@ task CollectRawWgsMetrics {
       OUTPUT=~{metrics_filename} \
       USE_FAST_ALGORITHM=true \
       READ_LENGTH=~{read_length}
+
+    sed 's/picard.analysis.WgsMetrics/picard.analysis.CollectWgsMetrics\$WgsMetrics/' ~{metrics_filename}  > ~{metrics_filename}.fixed
+
+    mv ~{metrics_filename}.fixed ~{metrics_filename}
```
Comment (tmooney) [inline tasks/Qc.wdl]: Could use `sed`'s `-i` option to avoid this step if you wanted.

File: tasks/Qc.wdl (hunk 1 above)
Comment (jasonwalker80) [reply tasks/Qc.wdl]: oops, I'll fix.

File: tasks/Qc.wdl (hunk 2 above)
Comment (jasonwalker80) [reply tasks/Qc.wdl]: thanks! that was one reason for making the PR, I figured there was a better way.

**Pull request merged**